### DISCUSSION TITLE
【Keyball61_右側トラックボールあり】パネル取り外し時にキーソケットのランドが剥がれてしまいました

### DISCUSSION THREAD

**Discussion by JinBallMen** [Yowkees/keyball#288] [Q&A]
以下、一度アクリルパネル等を取り付けましたが
一度取り外したくなりキースイッチを引き抜き工具で抜きました。
その際に、キーソケットを取り付ける部分の銀箔部分ごと剥がれてしまいました。

こうなると、修復はエナメル線で繋げますでしょうか。
また、剥離した箇所の少し上のダイオードに繋ぐ際は一つの線でよろしいでしょうか。

![20230727_090644~2](https://github.com/Yowkees/keyball/assets/75989686/04eb90b2-acac-4aaf-a96e-8983c7dae917)

**Comment by JinBallMen** [comment]
回路図を見て日が浅いのでご教示頂きたいのですが

すぐ下側のキーソケットも動かなくなっておりまして
rev1の回路図を見るとCOL3は下側のキーソケットとも繋がっているかと思うのですが、rev2でも同様でしょうか。

また、ROW4についてはダイオードを経由して繋がっているかと思っております。

以下画像の剥離した位置にキーソケットを置いていると仮定した際
リカバリ方法は以下のように繋げれば良いでしょうか。

![Polish_20230728_075652673](https://github.com/Yowkees/keyball/assets/75989686/3260ddf5-e50a-4aa0-8586-08a71f8a69fc)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ちょっと一部違います。
下記の同色どうしをエナメル線等で接続いただければ幸いです。緑点は３か所ありますので、３点がすべて導通するように接続お願いします。
なお、キーソケットの保持力がなくなりますので、キーソケットを両面テープ等で固定することをお勧めします。

以上、お手数ですがご対応お願いいたします。
![kb61_hagarefix](https://github.com/Yowkees/keyball/assets/84209772/b1516eac-4179-42bb-b653-9e540446a30a)

**Comment by JinBallMen** [comment]
無事つながりました、ありがとうございます。
また、トラックボールが動かない事象があるので、一旦原因探っておりますが
もし解決できそうにない場合は、また別途質問させていただきます。
